kubeflow/trainer/rhai/transformers.py

```
@@ -323,9 +331,151 @@ def checkpoint_in_progress(self):
+                except Exception as e:
+                    raise RuntimeError(
+                        f"Failed to check this node has access to the storage path: "
+                        f"'{cloud_remote_storage_uri}'. Error: {e}. "
+                        f"If using self-signed certificates, "
+                        f"set verify_cloud_storage_ssl=False. "
+                        f"If experiencing permission issues, check you have read and "
+                        f"write permissions to '{cloud_remote_storage_uri}'. "
+                        f"This check can be disabled by setting verify_cloud_storage_access=False."
+                    ) from e
+
+        def on_init_end(self, args, state, control, **kwargs):
```

Ruff (0.14.14)

Unused method argument: `control`

(ARG002)